woocommerce/woocommerce-accommodation-bookings
# Add product unit filter to add night unit support

### All Submissions:

### Changes proposed in this Pull Request:

Filter product unit to add night unit support.


Closes #320.

### How to test the changes in this Pull Request:

1. Create an Accommodations Booking Product.
2. Set the minimum duration to two or more nights, the maximum to a short range as well.
3. Navigate to the front end and view your product
4. Select a number of days under or over the allowed range
5. Note that the message refers to the duration_unit as "Nights", not "Blocks".

### Changelog entry

> Dev - Add product unit filter, `wc_bookings_product_duration_fallback`, to add night unit support.
